core/src/HIP/Kokkos_HIP_UniqueToken.hpp

```diff
@@ -45,9 +45,12 @@
 #ifndef KOKKOS_HIP_UNIQUE_TOKEN_HPP
 #define KOKKOS_HIP_UNIQUE_TOKEN_HPP
 
-#include <impl/Kokkos_ConcurrentBitset.hpp>
+#include <Kokkos_Macros.hpp>
+#ifdef KOKKOS_ENABLE_HIP
```

Comment:
This is not needed. If HIP is not enabled, CMake will ignore the file

Reply:
removed